README.md

```diff
@@ -32,16 +35,30 @@ Initial support is offered for the following (with more to follow soon):
 - LLMs
   - An adapter is provided for Steamship's OpenAI integration (`steamship_langchain.llms.OpenAI`)
   - An adapter is provided for *caching* LLM calls, via Steamship's Key-Value store (`SteamshipCache`) 
+- Document Loaders
+  - An adapter for loading Steamship Files into Documents is provided (`steamship_langchain.document_loaders.SteamshipLoader`)
```

Comment:
Is it Files -> Documents or Documents -> Files?  Or both?

Reply:
Files -> Documents.

Reply:
maybe i should say `exporting` ?